core/src/main/java/dk/alexandra/fresco/logging/PerformanceLogger.java

```diff
@@ -12,24 +12,26 @@
  *
  */
 public interface PerformanceLogger {
-
-  public Logger log = LoggerFactory.getLogger(PerformanceLogger.class);
-  public DecimalFormat df = new DecimalFormat("#.00");
   
-  public enum Flag {
-    LOG_NETWORK, LOG_RUNTIME, LOG_NATIVE_BATCH;
-
-    public static final EnumSet<Flag> ALL_OPTS = EnumSet.allOf(Flag.class);
-  }
-
   /**
-   * Prints any performance numbers picked up.
+   * Resets any counters/maps/lists used. 
    */
-  public abstract void printPerformanceLog(int myId);
+  public abstract void reset();    
```

Comment:
A public interface have by default public abstract methods - do not declare them.